analytics/readme.md

```diff
@@ -0,0 +1,33 @@
+
+Compiler Argument Statistics based on Google Analytics
+======================================================
+
+This little script will gather Analytics data from Google
+for Compilation events and which compiler arguments were used.
+
+Based on the amount of times the arguments were used, this
+data will be stored in an Compiler Explorer S3 bucket for later usage.
+
+We can use this data to suggest commonly used Compiler arguments to users.
+
+
+On the Admin server
+===================
+
+Run `sh install.sh` first
+
+
+How to run
```

Comment:
Thanks for the explanation. To be useful we'll probably need to automate this somehow, else we'll get out of date pretty quickly.